sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -183,16 +202,21 @@ object ScalaJSPlugin extends AutoPlugin {
 
         scalaJSLinkerConfig := StandardConfig(),
 
+        scalaJSLinkerImpl := LinkerImpl.default(getClass().getClassLoader()),
```

Comment:
What's the point of loading reflectively if you are using the current class loader?